dashboard/config/scripts/ECSPD-NexTech.script

```diff
@@ -0,0 +1 @@
+ECSPD.script
```

Comment:
this is how git does symlinks -- when this file gets pulled it will be a symlink